arch/arm/boot/dts/overlays/vc4-kms-dpi-hyperpixel2r-overlay.dts

```diff
@@ -0,0 +1,110 @@
+/*
+ * vc4-kms-dpi-hyperpixel2r-overlay.dts
+ */
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/bcm2835.h>
+
+/dts-v1/;
+/plugin/;
+
+/ {
+	fragment@0 {
```

Comment:
This needs a `compatible` string.

Reply:
Done. It originally included vc4-kms-dpi-hyperpixel.dtsi which did it for me.